packages/block-library/src/quote/index.js

```diff
@@ -233,6 +233,14 @@ export const settings = {
 		);
 	},
 
+	merge( attributes = {}, attributesToMerge = {} ) {
+		return {
+			...attributes,
+			value: ( attributes.value || [] ).concat( attributesToMerge.value || [] ),
```

Comment:
- Why not concat like we did in the other block?
- Is a `value` ever not an array?

Reply:
> Why not concat like we did in the other block?

The reason for the difference between the quote and other blocks is because the quote can contain multiple paragraphs (fields of text). So for the quote, I think we should concat to the existing list of paragraphs.

> Is a value ever not an array?

I think the value is always an array but as I was in doubts and I kept the checks. Now I did some tests with code assuming the value is always an array and things worked great so I think it is safe to assume the value is always an array (and it should be unless there is a bug in another part).